src/app/bitcoin/transactions.ts

```diff
@@ -57,7 +57,7 @@ export const useTransactions = (api: AxiosInstance): TxInstance => {
   };
 
   const postTx = async (params: { txhex: string }) => {
-    const { data } = await api.post<string>(`/tx`, { txhex: params.txhex });
+    const { data } = await api.post<string>(`/tx`, params.txhex);
```

Comment:
this line is the change?

Reply:
yes, need to submit the tx hex as the body rather than wrapped in json object

the rest is just some testing changes